scipy/special/cdf_wrappers.c

```diff
@@ -29,25 +29,25 @@
  */
 
 #define CDFLIB_CALL2(func, name, a, b, result, return_bound)            \
-    if (npy_isnan(p) || npy_isnan(q) || npy_isnan(a) || npy_isnan(b) || \
-        npy_isnan(bound)) {                                             \
-        return NPY_NAN;                                                 \
+    if (isnan(p) || isnan(q) || isnan(a) || isnan(b) || \
+        isnan(bound)) {                                             \
+        return NAN;                                                 \
     }                                                                   \
     func(&which, &p, &q, &a, &b, &status, &bound);                      \
```

Comment:
Would be nice to keep the formatting of the macros intact.
```suggestion
    if (isnan(p) || isnan(q) || isnan(a) || isnan(b) || isnan(bound)) { \
        return NAN;                                                     \
    }                                                                   \
    func(&which, &p, &q, &a, &b, &status, &bound);                      \
```

(same below)